src/Thoth.Json.Net.fsproj

```diff
@@ -8,6 +8,7 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageTags>fable;fable-library;fable-dotnet;fsharp;json</PackageTags>
     <Authors>Maxime Mangel</Authors>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
❓ Was there any particular reason to switch from Standard to .NET 6? As a package, Standard would give consumers more flexibility. (I think we can safely drop framework 4.6 though.)

Reply:
That's a good call-out; I wasn't really thinking but yeah targeting standard makes more sense 👍